drivers/flash/nrf_qspi_nor.c

```diff
@@ -466,23 +467,17 @@ static int qspi_nrfx_configure(const struct device *dev)
 		/* If quad transfer was chosen - enable it now */
 		if ((qspi_is_used_write_quad_mode(QSPIconfig.prot_if.writeoc))
 		    || (qspi_is_used_read_quad_mode(QSPIconfig.prot_if.readoc))) {
-
-			/* WRITE ENABLE has to be sent before QUAR ENABLE */
-			struct qspi_cmd cmd = { .op_code = SPI_NOR_CMD_WREN };
-
-			if (qspi_send_cmd(dev, &cmd) != 0) {
-				return -EIO;
-			}
-
 			uint8_t tx = BIT(CONFIG_NORDIC_QSPI_NOR_QE_BIT);
-
-			const struct qspi_buf tx_buff = { .buf = &tx, .len = sizeof(tx), };
-
-			cmd.op_code = SPI_NOR_CMD_WRSR;
-			cmd.tx_buf = &tx_buff;
-			cmd.rx_buf = NULL;
-
-			if (qspi_send_cmd(dev, &cmd) != 0) {
+			const struct qspi_buf tx_buff = {
+				.buf = &tx,
+				.len = sizeof(tx),
+			};
+			struct qspi_cmd cmd = {
```

Comment:
Doesn't need to add `.rx_buff = NULL` here?

Reply:
No; unspecified structure fields are default-initialized to zeros.